docs/why-netdata/1s-granularity.md

```diff
@@ -0,0 +1,80 @@
+# 1s granularity
+
+High resolution metrics are required to effectively monitor and troubleshoot
+systems and applications.
+
+## why?
```

Comment:
The subsection titles should start with a capital letter.

Reply:
yeap